index.js

```diff
@@ -156,11 +160,27 @@ const serializers = {
     )} in ${toUrlFormat(item.repo.name)}`;
   },
   PullRequestEvent: (item) => {
-    const emoji = item.payload.action === "opened" ? "💪" : "❌";
-    const line = item.payload.pull_request.merged
-      ? "🎉 Merged"
-      : `${emoji} ${capitalize(item.payload.action)}`;
-    return `${line} PR ${toUrlFormat(item)} in ${toUrlFormat(item.repo.name)}`;
+    let emoji = "";
+    let actionText = "";
+
+    // Check action type to determine emoji and text
+    // Note: GitHub Events API returns "merged" as an action type
+    if (item.payload.action === "opened") {
+      emoji = "💪";
+      actionText = "Opened";
+    } else if (item.payload.action === "closed") {
+      emoji = "❌";
+      actionText = "Closed";
+    } else if (item.payload.action === "merged") {
+      emoji = "🎉";
+      actionText = "Merged";
+    } else {
+      // Fallback for other actions
+      emoji = "⚡";
+      actionText = capitalize(item.payload.action);
+    }
```

Comment:
instead of this else statement initialize the variables with the content at the top

```suggestion
    }
```